x/auth/signing/handler_map.go

```diff
@@ -0,0 +1,59 @@
+package signing
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/cosmos/cosmos-sdk/types/tx"
+)
+
+// SignModeHandlerMap is SignModeHandler that aggregates multiple SignModeHandler's into
+// a single handler
+type SignModeHandlerMap struct {
```

Comment:
```suggestion
type MultiSignModeHandler struct {
```

Reply:
I'd rather not use `MultSign` here as this could be confused with `MultiSig`.  How about `CompositeSignModeHandler` or something like that?

Reply:
Mehhh, we can leave the name as-is -- it's fine.